README.md

```diff
@@ -25,3 +42,15 @@ We try not to close issues that have been answered adequately; leaving them open
 #### Rendering to a page
 
 All these questions and answers will be compiled into a webpage rendered at http://ipfs.io/docs/install (still TODO)
+
+## Contribute
+
+Feel free to join in. All welcome.
```

Comment:
It's a bit unclear what this means in the context of an FAQ collection, especially considering that how to ask and answer questions is already discussed above. It would be good to clarify this.